tests/sdk/nodejs/examples/helm-api-versions/step1/helm-single-api-version/Chart.yaml

```diff
@@ -1,3 +1,4 @@
+apiVersion: v2
```

Comment:
should these be v3?

Reply:
Apparently this is correct. `v1` corresponds to Helm v2, and `v2` corresponds to Helm v3. 🤷‍♂️